stdlib/LinearAlgebra/src/lbt.jl

```diff
@@ -13,6 +13,18 @@ struct lbt_library_info_t
     f2c::Int32
     cblas::Int32
 end
+
+macro get_warn(map, key)
+    return quote
+        if !haskey($(esc(map)), $(esc(key)))
+            @warn(string("[LBT] Unknown key into ", $(string(map)), ": ", $(esc(key)), ", defaulting to :unknown"))
+            $(esc(map))[-1]
```

Comment:
Maybe for consistency something like (probably need to interpolate the value to avoid issues using this macro in other modules):
```suggestion
            $(esc(map))[LBT_INTERFACE_UNKNOWN]
```
?